modules/govuk/manifests/node/s_search.pp

```diff
@@ -20,7 +20,7 @@
   }
 
   # Set Plek for AWS to Carrenza communication
-  if ( ( $::aws_migration == 'search' ) and ($::aws_environment == 'staging') ) or ( ($::aws_migration == 'search' ) and ($::aws_environment == 'production') ) {
+  if ( ($::aws_migration == 'search' ) and ($::aws_environment == 'production') ) {
     $app_domain = hiera('app_domain')
 
     govuk_envvar {
```

Comment:
Same as above; I think this is changing the URI for `email-alert-api` when it shouldn't be.